.travis.yml

```diff
@@ -1,17 +1,20 @@
-sudo: true
-
+os: linux
+dist: xenial
 language: cpp
 
 services:
   - docker
 
 before_install:
-  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/install-package.sh
-  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/build-package.sh
+  - openssl aes-256-cbc -K $encrypted_3959f4b55449_key -iv $encrypted_3959f4b55449_iv
+    -in ./deploy_key.enc -out ./deploy_key -d
+  - export SSH_KEY_PRIVATE=$(cat deploy_key)
```

Comment:
Is there a need to export it? You can also read it where you need it.

Reply:
Fixed.